README.md

```diff
@@ -4,6 +4,8 @@ The `datadog-connector` module provisions a Kinesis Firehose which is configured
 
 To provide the Sym Runtime with access to the resources created in this `Connector`, use the `aws/kinesis-firehose` addon in the `runtime-connector` module.
 
+For EU Datadog, make sure to set the `datadog_intake_url` input to `https://aws-kinesis-http-intake.logs.datadoghq.eu/v1/input`!
```

Comment:
I can't seem to find in the docs where other URLs might be besides US and EU 🤔 but if you can find it, that might be good to include?

Reply:
Based on https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination/?tab=kinesisfirehosedeliverystream#overview there seems to be only 2 regions "US" and "EU" for firehose